### ISSUE TITLE
clamav 0.104.1 Could NOT find CURSES - using ncursesw (wide-character)

### ISSUE THREAD

**Issue by Devstellar** [Cisco-Talos/clamav#353]
Clamav 0.104.1 fails to detect CURSES when ncursesw is installed (wide-character).
However Clamav 0.104.0 detects CURSES with same ncursesw installed.

This issue is somehow related to #292 

First of all, in ncurses:
```
 --enable-widec

    This switch causes wide-character libraries (e.g., libncursesw.so.6.2) to be built instead of normal ones (e.g., libncurses.so.6.2). These wide-character libraries are usable in both multibyte and traditional 8-bit locales, while normal libraries work properly only in 8-bit locales. Wide-character and normal libraries are source-compatible, but not binary-compatible.
```
Also:
```
# cat /usr/lib64/libncurses.so
INPUT(-lncursesw)
```
So, basically systems has ncurses.so that redirects to ncursesw as ncursesw can handle "normal" curses calls.

0.104.0 detects library without problem (clamdtop working). Logs:
-- Found CURSES: /usr/lib64/libncurses.so

0.104.1 gives error. Logs:
CMake Error at /usr/share/cmake-3.18/Modules/FindPackageHandleStandardArgs.cmake:165 (message):
  Could NOT find CURSES (missing: CURSES_LIBRARY)
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FindPackageHandleStandardArgs.cmake:458 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindCURSES.cmake:65 (find_package_handle_standard_args)
  CMakeLists.txt:490 (find_package)

-- Configuring incomplete, errors occurred!
See also "/compile/clamav-0.104.1/build/CMakeFiles/CMakeOutput.log".
See also "/compile/clamav-0.104.1/build/CMakeFiles/CMakeError.log".

**Comment by micahsnyder** [comment]
This works for me.  I also confirmed `clamdtop` works if you remove `ncurses` and just use `ncursesw` on a box with both.

**Comment by micahsnyder** [comment]
This issue should be resolved by:
https://github.com/Cisco-Talos/clamav/pull/355
https://github.com/Cisco-Talos/clamav/pull/364

The fixes have also been backported for the 0.104.3 patch release here: 
https://github.com/Cisco-Talos/clamav/pull/480
